src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -19,15 +22,24 @@ public void canGet() {
 
   @Test
   public void canPutAndGet() {
```

Comment:
Given that this change is to stop clients from defining rules with IDs that do not refer to existing policies, I would expect a test to demonstrate that behaviour.

Is that in a different test class?

Reply:
yes, the test that demonstrate the behavior with wrong policy is here:
CheckOutByBarcodeTests#cannotUpdateCirculationRulesWithInvalidNoticePolicyId

Probably it would be better if I move it to CirculationRulesAPITests....

Reply:
Thanks. Yes, please move it to be with the tests about changing the rules.

Are there tests for each policy type?

Reply:
The test has been moved to CirculationRulesAPITests and tests for each policies have been added as well.

Reply:
Thank you